### PR TITLE
Update hero copy and link

### PR DESCRIPTION
**Pull request by markpeterbuckley**
Changes strapline, added additional link to find out more

### CODE REVIEW

File: components/MastHead/index.tsx [hunk 1]
```diff
@@ -8,12 +8,12 @@ const Masthead = () => {
...
         <div className="govuk-grid-row">
           <div className="masthead__body govuk-grid-column-two-thirds">
-            <h1 className="govuk-heading-xl masthead__title ">Host your service in the Cloud< span className="govuk-visually-hidden">with GOV.UK Platform as a service (PaaS)</span></h1>
+            <h1 className="govuk-heading-xl masthead__title ">Host your applications in the Cloud< span className="govuk-visually-hidden">with GOV.UK Platform as a service (PaaS)</span></h1>
             <div className="masthead__inline-image">
               <img src="/assets/images/paas-top-image.png" alt="" role="presentation" />
             </div>
             <p className="govuk-!-margin-bottom-5 masthead__description">
-              If you work in the public sector, use GOV.UK PaaS to host your service without worrying about infrastructure.
+              If you work in the public sector, use GOV.UK Platform as a Service to host your services without worrying about infrastructure.
```
Comment (0atman) [inline components/MastHead/index.tsx:16]: as line 11, lets call these "applications" right?

**Review by 0atman — CHANGES_REQUESTED** [review]

**Review by kr8n3r — COMMENTED** [review]
@SaraC19 do we update the event labels as well?

**Comment by SaraC19** [comment]
@kr8n3r  - when this goes live, yes please to update the event label to the new words

File: components/MastHead/index.tsx (hunk 1 above)
Comment (LeePorte) [reply components/MastHead/index.tsx:16]: Agree

**Comment by kr8n3r** [comment]
superceded by https://github.com/alphagov/paas-product-pages/pull/130 and these suggestions included

**Pull request closed without merging**